crates/type-system/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(lint_reasons)]
+mod ontology;
+mod utils;
+
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
+// allocator.
```

Comment:
Not really a fan of this comment as this is exactly what is written in the code below

Reply:
Happy to delete it, the comment was from the boilerplate when the project was generated

Reply:
Have deleted